File: ccc/2018/j2.java
//https://dmoj.ca/problem/ccc18j2
//CCC '18 J2 - Occupy parking

import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
		int n = s.nextInt();
		int out = 0;
		String yes = s.next();
		String tod = s.next();
		for (int i=0; i<n; i++) {
			if (yes.toCharArray()[i] == 'C' && tod.toCharArray()[i] == 'C') {
				out++;
			}
		}
    	System.out.println(out);
		s.close();
    }
}

File: ccc/2018/j2.py
#https://dmoj.ca/problem/ccc18j2
#CCC '18 J2 - Occupy parking

N = int(input())
y = input()
t = input()

x = 0

list(y)
list(t)

for a, b in zip(y, t):
    if a == b == "C":
        x += 1
      
print(x)
